Client/mods/deathmatch/logic/luadefs/CLuaElementDefs.cpp

```diff
@@ -1551,6 +1554,29 @@ int CLuaElementDefs::IsElementFrozen(lua_State* luaVM)
     return 1;
 }
 
+int CLuaElementDefs::IsElementOnGround(lua_State* luaVM)
```

Comment:
Replace this with:
```cpp
bool CLuaElementDefs:::IsElementOnGround(CEntity* entity) {
    switch (Entity.GetType())
    {
    case CCLIENTPLAYER:
    case CCLIENTPED:
        return static_cast<CClientPed*>(entity)->IsOnGround();
    case CCLIENTVEHICLE:
        return static_cast<CClientVehicle*>(entity)->IsOnGround();
    default:
        throw std::invalid_argument{"Element type not supported"}; // TODO: Add `InvalidElementType{entity}` or something similar for the parser (so the exact type can be reported)
    }
}
```
And get rid of the function in `CStaticFunctionDefinitions`